tests/integration/standard/test_cluster.py

```diff
@@ -69,7 +56,8 @@ def test_ignored_host_up(self):
 
         @test_category connection
         """
-        ingored_host_policy = IgnoredHostPolicy(["127.0.0.2", "127.0.0.3"])
+        ingored_host_policy = HostFilterPolicy(RoundRobinPolicy(),
```

Comment:
It's a small thing, but I think this should be left as is, just to avoid one potential source of bugs